docs/UserGuide.adoc

```diff
@@ -138,6 +139,25 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+==== Deleting a Tag (or multiple Tags)
+
+Deletes the specified tag from all people containing the tag in the address book. +
+Format: `delete [t/TAG]...`
+
+****
+* You can delete multiple tags with one delete command.
+* The tag (or all the tags) specified must already exist in the address book.
+****
+
+Examples:
+
+* `list` +
+`delete t/friends` +
```

Comment:
Good to specify if the command is case-sensitive regarding the tags?

Reply:
Good point. Made a commit. Please review